sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncReadBenchmark.java

```diff
@@ -48,24 +48,25 @@ protected void hookOnError(Throwable throwable) {
 
     AsyncReadBenchmark(Configuration cfg) {
         super(cfg);
+        cosmosAsyncContainer = v4Client.getDatabase(cfg.getDatabaseId()).getContainer(cfg.getCollectionId()).read().block().getContainer();
```

Comment:
Why are we reading the container again here ?
When calling `super(cfg)`, this will read the container in `AsyncBenchmark` constructor on line 75. 
If we change AsyncBenchmark to completely use `v4Client` then we can reuse the container read in the parent class. Is this PR only reflects partial changes for changing the clients, are there going to be more PRs which will handle the complete set of changes?

Reply:
the parent class and other benchmarks are using documentClient. This PR will only change the AsyncReadBenchmark. The other benchmarks will be updated later, specifically we also have query api changes coming. So all those other benchmarks will be updated later once the surface API is frozen.

and on line 75 of the parent it reads the collection not container. here we need container.

Reply:
Sounds good for now. 
Can we put a TODO before this container read statement specifying `Get rid of this container read once all the benchmark uses v4Client`
Because once we change all the benchmarks, we should make sure not to re-read the container, since at that point, we have had already read the container through v4 client in parent class. 
Makes sense ?

Reply:
good to have. comment added.